models/demos/resnet/tt/metalResnetBlock50.py

```diff
@@ -2314,8 +2314,13 @@ def forward(self, x: tt_lib.tensor, write_event=None, op_event=None, final_out_m
                 output_dtype=self.model_config["ACTIVATIONS_DTYPE"],
             )
         else:
-            x = tt_lib.tensor.pad(
-                x, padded_shape, [0, 0, 0, 0], 0, output_mem_config=self.memory_config, use_multicore=True
+            x = ttnn.pad(
+                x,
+                ttnn.Shape(padded_shape),
+                ttnn.Shape([0, 0, 0, 0]),
+                0,
+                output_mem_config=self.memory_config,
+                use_multicore=True,
```

Comment:
why do we need `use_multicore`?

Reply:
initially I thought no models 'use_multicore=False' but many do as that is the default and many models didn't change it to true. 
When I tried changing it (for example in the sweep tests) there were some asserts that broke so for now we will have to leave it in (to keep parity with tt-lib )